engine/src/main/resources/org/camunda/bpm/engine/db/upgrade/db2_engine_7.13_to_7.14.sql

```diff
@@ -25,3 +25,7 @@ values ('telemetry.lock', '0', 1);
 -- insert telemetry.lock in property table - https://jira.camunda.com/browse/CAM-12031  --
 insert into ACT_GE_PROPERTY
 values ('installationId.lock', '0', 1);
+
+-- quick sorting of incidents - https://jira.camunda.com/browse/CAM-12383
+create index ACT_IDX_HI_INCIDENT_CREATE_TIME on ACT_HI_INCIDENT(CREATE_TIME_);
+create index ACT_IDX_HI_INCIDENT_END_TIME on ACT_HI_INCIDENT(END_TIME_);
```

Comment:
If the indexes should be rolled out with 7.13 and 7.12, they need dedicated patch scripts and should not be included in the update script for the minor release (compare the changes in https://github.com/camunda/camunda-bpm-platform/pull/966). The role of the minor update script is essentially: Update from the latest patch level to 7.14.0. 

Please just revert your changes to the minor release update scripts here. I will take care of the patch scripts and the backport after we merge https://github.com/camunda/camunda-bpm-platform/pull/966).